server/lib/src/idm/server.rs

```diff
@@ -1979,7 +2024,11 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
             .get_changed_uuids()
             .contains(&UUID_SYSTEM_CONFIG)
         {
+            // TODO: probably it would be more efficient to introduce a single check for each of the possible system configs
+            // but that would require changing what uuid the operation is assigned
             self.reload_password_badlist()?;
+            self.reload_authsession_expiry()?;
```

Comment:
Could these all be coallesced into a single "reload_system_account_policy"?

Reply:
Actually, yes 😄